### PR TITLE
Web client URLs from MARC exporter integration.

### PR DESCRIPTION
**Pull request by aslagle**
The circulation piece of https://github.com/NYPL-Simplified/server_core/pull/1002

### CODE REVIEW

File: api/marc.py [hunk 1]
```diff
@@ -41,21 +41,32 @@ def annotate_work_record(self, work, active_license_pool, edition,
...
+
         from api.registry import Registration
-        settings = _db.query(ConfigurationSetting).filter(ConfigurationSetting.key==Registration.LIBRARY_REGISTRATION_WEB_CLIENT, ConfigurationSetting.library_id==library.id)
+        settings += [s.value for s in _db.query(
+            ConfigurationSetting
+        ).filter(
+            ConfigurationSetting.key==Registration.LIBRARY_REGISTRATION_WEB_CLIENT,
+            ConfigurationSetting.library_id==library.id
+        ) if s.value]
+
```
Comment (leonardr) [inline api/marc.py]: Is this a different way of writing the same code as before? It looks like it to me but I want to make sure.

Comment (leonardr) [reply api/marc.py]: I guess it lets you filter out empty values earlier.

**Comment by leonardr** [comment]
This looks good. If you decide to go up one level of abstraction with something analogous to  MirrorUploader, I'm thinking that object would replace `integration`.

File: api/marc.py (hunk 1 above)
Comment (aslagle) [reply api/marc.py]: Yeah, it's pretty much the same, just reorganized.

**Pull request merged**